src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length">
+      <div class="calendar__day calendar__day--first"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length calendar__day--first">
      <div class="calendar__day"></div>
```
here I should be able to pass certain month length: 28, 29, 30, 31 and certain starting day: mon, tue, wed ...
you should be able to modify them in devtools, according to README, and your calendar should behave correctly